src/test/pow_tests.cpp

```diff
@@ -11,112 +11,40 @@
 
 BOOST_FIXTURE_TEST_SUITE(pow_tests, BasicTestingSetup)
 
+static const std::vector<std::pair<uint32_t, uint32_t>> g_mainnet_data = {
```

Comment:
nit: no need to pollute global namespace, it's a local variable for this test.

Reply:
Resolved in latest push